esp/platform/txsummary.hpp

```diff
@@ -78,9 +91,12 @@ class CTxSummary : extends CInterface
     };
 
     using Entries = std::list<Entry>;
+    using Timers = std::map<StringBuffer, CumulativeTimer*>;
 
-    Entries   m_entries;
-    unsigned  m_creationTime;
+    mutable Monitor m_sync;
```

Comment:
minor: The platform code generally used a CriticalSection/CriticalBlock for a mutex which has no timeout.  It is lighter weight (especially on windows) than a mutex.